config/500-controller.yaml

```diff
@@ -42,7 +42,12 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        - name: CONFIG_LOGGING_NAME
```

Comment:
Perhaps I'm missing something here, but the way I understand this change to be is:
you must have a configmap called eventing-config-logging, which must contain a key called configMapName that could point to another configmap which actually contains the logging configuration. If that's the case, then why not just take in a env string here instead of a deref through a configmap? I think that would be cleaner and simpler. But again, perhaps I misunderstood something.

Reply:
I just want to make this env value looks to be related with the `configmap` name.  But you are right, just a env string here would be cleaner and simpler, I update here.